packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -485,7 +486,11 @@ const serviceEntityPage = (
       <EntityKafkaContent />
     </EntityLayout.Route>
 
-    <EntityLayout.Route path="/todos" title="TODOs">
+    <EntityLayout.Route
+      path="/todos"
+      title="TODOs"
+      permission={catalogEntityDeletePermission}
```

Comment:
Bit of a strange example this, is it not? 😅

Reply:
Yes, definitely 😅 I was just using it for testing, I should remove it or come up with a better example.